### PR TITLE
LPS-145050

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina @tinatian
I just made a new issue on this.
this is cleaned up from: #2473, continued on top of your pull here: julschong#68
Thank you!

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

File: portal-impl/src/com/liferay/portal/util/PortalClassPathUtil.java [hunk 1]
```diff
@@ -56,16 +59,23 @@ public static ProcessConfig createProcessConfig(Class<?>... classes) {
 
 		File[] files = _listClassPathFiles(classes);
 
-		StringBundler classpathSB = new StringBundler(files.length * 2);
+		if (files.length == 0) {
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/util/PortalClassPathUtil.java:62]: @julschong The changes in this method are not related to the commit message of this pull, please merge this part of change in to last commit.

Comment (tinatian) [reply portal-impl/src/com/liferay/portal/util/PortalClassPathUtil.java:62]: @julschong Please note one thing, that the throwing IllegalStateException is actually a change to the old code. If  you merge the code into last commit, you should not merge in this part. This should be separate commit.

**Comment by tinatian** [comment]
@julschong Please see my comments above, if you have any question about it. Let's have a talk.

Tina.

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by tinatian** [comment]
Talked with @julschong , please resend.

Thanks.
Tina.
